app/catalog_data.go

```diff
@@ -8,22 +8,24 @@ import (
 )
 
 const (
-	defaultURL = "https://github.com/ibuildthecloud/rancher-charts"
+	defaultURL    = "https://github.com/rancher/rancher-charts"
```

Comment:
Okay, who needs to be shot?  rancher/rancher-charts..... How about rancher/charts.

Reply:
I just renamed the repo to `rancher/charts` so PR should be updated @StrongMonkey